packages/forge2d/lib/src/dynamics/world.dart

```diff
@@ -149,24 +149,24 @@ class World {
     bodies.remove(body);
   }
 
-  /// Create a joint to constrain bodies together. No reference to the definition is retained.
+  /// Adds a joint to constrain bodies together.
```

Comment:
@spydon What do you think of this doc comment. The verb I'm using is "add" since the `Joint` has already been initialised?